integration-tests/test_simplest_plugin.py

```diff
@@ -46,3 +53,11 @@ def test_config_hook():
         data = yaml.load(f)
 
     assert data['simplest_plugin']['present']
+
+def test_tmpauthenticator():
+    """
```

Comment:
I would probably name this *test_tmpauthenticator_enabled* or *test_jupyterhub_config_hook* (I'm not very good at naming things though :eyes: ) but the name as it is makes me think we're testing the authenticator.

Reply:
Good catch! I've renamed it!